documentation/known_issues.markdown

```diff
@@ -43,9 +43,11 @@ To use a different temporary directory, you can set the following JVM property:
 ```
 
 When Puppet Server is installed from packages, this property should be added
```

Comment:
Suggested reword: "When Puppet Server is installed from packages, add this property to the...."